app/views/shared/_header.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= stylesheet_link_tag 'header' %>
```

Comment:
I think you can move it to application.html.erb because header is always rendered.

Reply:
I like it when all dependencies are specified where they are used